q2_diversity_lib/plugin_setup.py

```diff
@@ -243,3 +250,100 @@
         citations['lozupone2011unifrac'],
         citations['mcdonald2018unifrac']]
 )
+
+# ------------------------ Dispatch ------------------------
+plugin.methods.register_function(
+    function=q2_diversity_lib.alpha_passthrough,
+    inputs={'table': FeatureTable[Frequency]},
+    parameters={'metric': Str % Choices(alpha.METRICS['NONPHYLO']['UNIMPL'])},
+    outputs=[('vector', SampleData[AlphaDiversity])],
+    input_descriptions={'table': "The feature table containing the samples "
+                        "for which a selected metric should be computed."},
+    parameter_descriptions={'metric':
+                            'The alpha diversity metric to be computed.'},
+    output_descriptions={'vector': "Vector containing per-sample values "
+                                   "for the chosen metric."},
+    name="Alpha Passthrough (non-phylogenetic)",
+    description="Computes a vector of values for each samples in a feature "
+                "table using the scikit-bio implementation of a chosen alpha "
+                "diversity metric."
```

Comment:
This makes it sound like the return data is many vectors (one for each sample), which isn't the case.

```suggestion
    description="Computes a vector of values (one value for each sample in a feature "
                "table) using the scikit-bio implementation of the selected alpha "
                "diversity metric."
```